zipkin-common/src/test/scala/com/twitter/zipkin/storage/DependencyStoreSpec.scala

```diff
@@ -103,28 +103,25 @@ abstract class DependencyStoreSpec extends JUnitSuite with Matchers {
     val two = Endpoint(127 << 24 | 2, 9410, "trace-producer-two")
     val three = Endpoint(127 << 24 | 3, 9410, "trace-producer-three")
 
-    val trace = List(
-      Span(10L, "get", 10L, None, Some(1445136539256150L), Some(1152579L), List(
```

Comment:
There is one known issue in the spark job that it always aggregates *everything* in the DB, i.e. it does not filter out any spans by timestamps. I am looking to fix it in https://github.com/openzipkin/zipkin-dependencies-spark/issues/7. But that's not the problem here.

It is that the job takes no inputs, and always saves the computed result as-of _today_. That's why the tests with hardcoded _query time_ are going to fail.